### ISSUE TITLE
ERROR WCAG2AA

### ISSUE THREAD

**Issue by sdash-redhat** [sdash-redhat/testrepo#4]
ERROR WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2 html  The html element should have a lang or xml:lang attribute which describes the language of the document.
